app/views/about.html

```diff
@@ -1 +1,43 @@
-<p>This is the about view.</p>
+<div class="about">
+    <h1>Welcome to MoodCat.me</h1>
+    <p>
+        Hi there!
+        Welcome to MoodCat.me, the music platform of the future, available today.
+        Join us in the adventure of finding and enjoying music geared to you!
+        We try to understand you
+    </p>
+    <h2>How does it work?</h2>
+    <p>
+        Go to the homepage, and select what mood(s) you're in.
+        Once you click next, we recommend you a couple of rooms to check out.
+        Pick one you think you like, and you will join the room.
+        We'll start playing music for you, and you can chat with people in the same mood as you are!
+    </p>
+    <blockquote>
+        This song doesn't fit the room, why am I hearing this?
+    </blockquote>
+    <p>
+        We're constantly trying to improve the system. You can give us feedback to help us.
+        If you think a song fits the current mood, click the <i class="glyphicon glyphicon-thumbs-up"></i> button in the room.
+        This way, we know whe made the right recommendation. Similarly, if the song does not fit the room, click the <i class="glyphicon glyphicon-thumbs-down"></i> button.
+        We'll try to modify the system based on your feedback. If you want to help us even better, you can use the popup to modify the rating of the song to your vision.
+    </p>
+    <h3>We need your help!</h3>
+    <p>
+        Want to make MoodCat a better place?
+        You can help us improving Moodcat in a couple of ways.
+        All users can participate in our <a href="/rate-the-song">rating game</a>.
+        Listen to 30 seconds of music, and give your feedback on the song.
+        For every 5 songs you rate, you earn <strong>30</strong> points!
+    </p>
+    <p>
+        But there are even more ways to earn points. If you find a bug in MoodCat,
```

Comment:
But can be omitted, it is not a contraposition, but a continuation of the previous point.